docs/source/editor/prepub.rst

```diff
@@ -0,0 +1,99 @@
+Pre Publication
+===============
+The Pre Publication stage of the workflow takes you thorugh the last checks before publication, going through each of these final elements to check step by step.
+There are some key things to look out for in Pre Publication, outlined in bold below.
+
+1. View Metadata 
+----------------
+View Metadata, the first step in Pre Publication, is where editors can make final checks and alterations to author abstracts, titles, licences, article citation and names/contact details.
+It is important to ensure that the abstract and title in the metadata exactly match the abstract and title of the PDF file (often, authors change their abstract during revision and copyediting, so the original abstract needs updating before publication). 
+**The metadata abstract is displayed when you preview the XML file** (if you are using XML files) **in the Typesetting Plugin.**
+ 
+
+.. figure:: ../nstatic/gifprepub.gif
+
+If you copy and paste an abstract into the metadata, ensure you paste this in without formatting and, if necessary, use an HTML cleaning tool so that the abstract text matches the formatting of the rest of the XML text.
+**Use the preview XML file function in the Typesetting Plugin to check abstracts have updated correctly.**
+
+It is very important, for Open Access compliance purposes, that the licence selected in the metadata (which, by default, is CC BY-4.0) matches any licensing information given in the footer of the PDF file. 
+
+.. warning::
+   **If an author requests a licence other than CC BY-4.0, this must be flagged in advance to the typesetters.** 
```

Comment:
Perhaps reword this to make it less OLH specific. The Typesetters _should_ be looking at the metadata page when creating the articles which displays this information.

Reply:
Do you mean the bit about the licensing? Can I do so by just adding 'For OLH journals,....' the typesetters do use a standard template and we haven't provided them with different text for different licences (yet) so it would need to be flagged...

Reply:
We should avoid adding anything OLH specific at all. You could say “where something like licence is out of the ordinary please flag it to typesetter’s” or something without being OLH specific about the default license (not everyone uses CCBY as default for example).

Reply:
Ok, have updated